src/main/kotlin/gg/skytils/skytilsmod/features/impl/trackers/impl/DupeTracker.kt

```diff
@@ -166,7 +167,7 @@ object DupeTracker : Tracker("duped_items") {
             "ITEM_STASH" -> event.toolTip.add("§c§lStashed item: possibly duped")
             "ITEM_COMMAND", "ITEM_MENU" -> event.toolTip.add("§c§lSpawned by admin lol")
             "" -> return
-            else -> if (Skytils.config.showOrigin) event.toolTip.add("§7§lOrigin: $origin")
+            else -> if (Skytils.config.showOrigin) event.toolTip.add("§7§lOrigin: " + origin.ifEmpty { "Unknown" }.split('_').joinToString(" ").toTitleCase())
```

Comment:
unknown case never runs

Reply:
Pretty sure it did when i tested it, tho that may have been with my own code and not Sychics

Reply:
the unknown case still never runs

Reply:
I just want to make sure, wdym with it does not run?

Reply:
Do you mean ifBlank should be used instead?

Reply:
no I mean it never runs